charts/neon-proxy/templates/deployment.yaml

```diff
@@ -105,13 +105,13 @@ spec:
             - --certs-dir
             - /certs-extra
             {{- end }}
+            {{- range .Values.settings.endpointRpsLimits }}
+              --endpoint-rps-limit={{ . | quote }}
```

Comment:
This should be an item in the `command` list, but the list marker was removed in https://github.com/neondatabase/helm-charts/pull/52/files

Reply:
Thank you, added it